### DISCUSSION TITLE
How to attach camera to springarm

### DISCUSSION THREAD

**Discussion by whdns94** [Sixze/ALS-Refactored#107] [Q&A]
Go to the AAlsCharacterExample.cpp 'CreateDefaultSubject' spring arm. 
And I attached a camera.
It doesn't work. Is there a way I'm missing?
Or is there a good way? :)

**Comment by Sixze** [comment]
`UAlsCameraComponent` does not require a spring arm to work.

**Reply by whdns94** [reply]
AAlsCharacterExample::AAlsCharacterExample()
{
	SpringArm = CreateDefaultSubobject<USpringArmComponent>(TEXT("SpringArm"));
	Camera = CreateDefaultSubobject<UAlsCameraComponent>(TEXT("AlsCamera"));
	SpringArm->SetupAttachment(RootComponent);
	SpringArm->SetRelativeLocationAndRotation(FVector(.0f, .0f, 50.f), FRotator(-50.f, .0f, .0f));
	SpringArm->TargetArmLength = 1400.f;
	SpringArm->bEnableCameraLag = true;
	SpringArm->CameraLagSpeed = 3.0f;
	Camera->SetupAttachment(SpringArm, USpringArmComponent::SocketName);
	Camera->SetRelativeRotation_Direct({0.0f, 45.0f, 0.0f});
}
not working. 
I need a springarm function
Is this absolutely impossible?
